jenkins/Jenkinsfile

```diff
@@ -0,0 +1,148 @@
+def secrets = [
+  [path: 'secret/jenkins/github', engineVersion: 2, secretValues: [
+    [envVar: 'PRIVATE_REPO_GITHUB_TOKEN', vaultKey: 'private_token']]],
+]
+
+def configuration = [vaultUrl: "${env.VAULT_URL}",  vaultCredentialId: 'vault-app-role', engineVersion: 2]
```

Comment:
I don't see you using vault?

Reply:
Correct, I assumed it's needed for connection with GitHub. Removed.